zulip_bots/zulip_bots/lib.py

```diff
@@ -239,9 +240,13 @@ def extract_query_without_mention(message: Dict[str, Any], client: ExternalBotHa
     the stripped message with the bot's @mention removed.  Otherwise, it returns None.
     """
     mention = '@**' + client.full_name + '**'
-    if not message['content'].startswith(mention):
-        return None
-    return message['content'][len(mention):].lstrip()
+    extended_mention_regex = re.compile('^@\\*\\*.*\\|' + str(client.user_id) + '\\*\\*')
```

Comment:
I think this might be better spelled using `r'` (aka raw strings); you need less backslash-escaping that way.